ci-operator/step-registry/upi/install/gcp/upi-install-gcp-commands.sh

```diff
@@ -335,10 +335,21 @@ EOF
 
 gcloud deployment-manager deployments create ${INFRA_ID}-control-plane --config 05_control_plane.yaml
 
+## Determine name of master nodes
+# https://github.com/openshift/installer/pull/3713
+set +e
+fgrep -e '-master-0' 05_control_plane.py
+set -e
+if [[ "$?" == 0 ]]; then
+  MASTER='master'
+else
+  MASTER='m'
+fi
+
 ## Configure control plane variables
-MASTER0_IP=$(gcloud compute instances describe ${INFRA_ID}-m-0 --zone ${ZONE_0} --format json | jq -r .networkInterfaces[0].networkIP)
-MASTER1_IP=$(gcloud compute instances describe ${INFRA_ID}-m-1 --zone ${ZONE_1} --format json | jq -r .networkInterfaces[0].networkIP)
-MASTER2_IP=$(gcloud compute instances describe ${INFRA_ID}-m-2 --zone ${ZONE_2} --format json | jq -r .networkInterfaces[0].networkIP)
+MASTER0_IP=$(gcloud compute instances describe ${INFRA_ID}-${MASTER}-0 --zone ${ZONE_0} --format json | jq -r .networkInterfaces[0].networkIP)
```

Comment:
qq: is there no way to request the information from the deploymentmanager template that was deployed. i.e all the resources it created?

Reply:
There likely is, but that's not what is in our documentation. I'd like to make sure we are sticking to documentation as much as possible, since that's the point.